torch_xla/experimental/deprecation.py

```diff
@@ -21,5 +22,22 @@ def wrapped(*args, **kwargs):
   return wrapped
 
 
+def mark_deprecated(module, new: FN) -> FN:
+  """Decorator to mark a function as deprecated and map to new function.
+
+  Args:
+    module: current module of the deprecated function that is in. Assume current module name is X, you can use `from . import X` and pass X here.
+    new: new function that we map to. Need to include the path the new function that is in.
+
+  Returns:
+    Wrapper of the new function.
+  """
+
+  def decorator(func):
+    return deprecated(module, new, old_name=func.__name__)
```

Comment:
Since you have a reference to both the new and old functions here, does `func.__module__` work in place of `module`? No worries if it doesn't work

Reply:
Nice catch! Updated to __module__.